boolean-logic/BooleanLogic.js

```diff
@@ -35,6 +35,8 @@ module.exports = function(RED) {
 							+ node.config.inputCount + " topics received, resetting. Will not output new value until " + node.config.inputCount + " new topics have been received.");
 					node.state = {};
 					h.DisplayUnkownStatus();
+				} else if (keyCount == 1) {
+					node.status({ fill: "yellow", shape: "dot", text: "One of 2 topics recieved." });
```

Comment:
It looks to me that the check should be `keyCount != node.config.inputCount` and that the message wrongly states "2 topics" when the actual expected count is `node.config.inputCount`?

I'm no longer running a NodeRed instance so this is just from quickly reading up on the code.

Reply:
> It looks to me that the check should be `keyCount != node.config.inputCount` and that the message wrongly states "2 topics" when the actual expected count is `node.config.inputCount`?
> 
> I'm no longer running a NodeRed instance so this is just from quickly reading up on the code.

You are correct.  I was wrongly assuming that this control only takes 2 inputs.  I will update.

Reply:
I have removed the condition on the else, and updated the status text.